app/models/etl/dirty.rb

```diff
@@ -0,0 +1,30 @@
+class ETL::Dirty
+  def self.process(*args)
+    new(*args).process
+  end
+
+  def initialize(date:)
+    @date = date
+  end
+
+  def process
+    create_new_version_for_dirty_items
+  end
+
+private
+
+  attr_reader :date
+
+  def create_new_version_for_dirty_items
+    Dimensions::Item.dirty_before(date).find_in_batches do |items|
+      new_items = items.map(&:new_version)
```

Comment:
What about breaking this block into two other methods that describe, at a higher level, what is happening?
Something along the following lines?
```ruby
Dimensions::Item.dirty_before(date).find_in_batches do |items|
  new_items = create_new_items(items)
  import_content_details(new_items)
end